spec/inferno_template/patient_group_spec.rb

```diff
@@ -79,13 +98,13 @@ def run(runnable, inputs = {})
         response_body: resource.to_json
       )
 
-      result = run(test)
+      result = run(test, url: url)
```

Comment:
Since the access token input is required, you have to pass in that input as well in all the spec tests